server/src/main/java/org/elasticsearch/index/codec/vectors/ES813Int8FlatVectorFormat.java

```diff
@@ -168,10 +166,5 @@ public Map<String, Long> getOffHeapByteSize(FieldInfo fieldInfo) {
         public void close() throws IOException {
             reader.close();
         }
-
-        @Override
-        public Map<String, Long> getOffHeapByteSize(FieldInfo fieldInfo) {
-            return OffHeapByteSizeUtils.getOffHeapByteSize(reader, fieldInfo);
-        }
```

Comment:
Don't we still need these? Or does some magical delegation occur?

Reply:
yeah, you're right that we do need to delegate. If you look a few lines up you will see the delegating overload. I put the methods in different positions in the file between the branches, so it appears as a full removal here. :-(